community/front-end/ofe/tf/main.tf

```diff
@@ -64,7 +64,7 @@ module "service_account" {
 
 module "control_bucket" {
   source  = "terraform-google-modules/cloud-storage/google"
-  version = "~> 2.2"
+  version = "~> 3.0"
```

Comment:
Please remove this as (A) it's not relevant to the purpose of the PR (B) the release notes for 3.0 introduce breaking changes (B) 4.0 is the latest anyhow

Reply:
This was the smallest change I found which would get me past the following error during the "terraform validate" step in pre-commits:

```
│ Error: Failed to query available provider packages
│ 
│ Could not retrieve the list of available versions for provider
│ hashicorp/google: no available releases match the given constraints >=
│ 3.53.0, ~> 3.53, ~> 4.0, < 5.0.0
```

Is there something better I could do to avoid that error?

Reply:
I see your point. The most correct changes to allow TPG 4.x compatibility would be either:

- `~> 3.1` (4.x wasn't allowed in 3.0)
- `~> 4.0`

Since we should be testing these changes by running the OFE anyhow, I would just bite the bullet and do 4.0.

Reply:
Please create a task in this iteration to test the OFE prior to release.

Reply:
Thanks for the tip, I've updated the "cloud-storage" module to `~> 4.0`, and created the task to test OFE.